### PR TITLE
Pricing Components

### PR DESCRIPTION
**Pull request by giteshk11**
# Issue Being Addressed
#565 

Put the issue(s) # that your PR is addressing here. e.g. `#15`, `#29`

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

- [ ] Bug Fix
- [x] Refactor
- [x] New Feature
- [x] Update to Existing Feature
- [ ] Other (state below)

# Description
- Created Account Manage Page with new routes
- Created components for pricing 
- Refactor existing components

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.
1. Login into the app
2. Click on Account from the bottom navigation
3. Click on settings (gear icon) from the top right corner.
4. Click on the Manage Account option.
5. Click on Upgrade to Pro to see the bottom slider

# Screenshots/casts

localhost_8080_wait_signup=1 token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9 eyJodHRwOi8vc2NoZW1hcy54bWxzb2FwLm9yZy93cy8yMDA1LzA1L2lkZW50aXR5L2NsYWltcy9uYW1laWRlbnRpZmllciI6IjU3OSIsImV4cCI6MTYyMjQ3NzU4MCwiaXNzIjoic3R (2)
localhost_8080_wait_signup=1 token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9 eyJodHRwOi8vc2NoZW1hcy54bWxzb2FwLm9yZy93cy8yMDA1LzA1L2lkZW50aXR5L2NsYWltcy9uYW1laWRlbnRpZmllciI6IjU3OSIsImV4cCI6MTYyMjQ3NzU4MCwiaXNzIjoic3R (1)
localhost_8080_wait_signup=1 token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9 eyJodHRwOi8vc2NoZW1hcy54bWxzb2FwLm9yZy93cy8yMDA1LzA1L2lkZW50aXR5L2NsYWltcy9uYW1laWRlbnRpZmllciI6IjU3OSIsImV4cCI6MTYyMjQ3NzU4MCwiaXNzIjoic3RhZ2l
localhost_8080_wait_signup=1 token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9 eyJodHRwOi8vc2NoZW1hcy54bWxzb2FwLm9yZy93cy8yMDA1LzA1L2lkZW50aXR5L2NsYWltcy9uYW1laWRlbnRpZmllciI6IjU3OSIsImV4cCI6MTYyMjQ3NzU4MCwiaXNzIjoic3R (3)

# Additional Context

Provide any additional notes that you think would help effective review of your code.

### CODE REVIEW

File: src/components/ExpansionPanel.vue [hunk 1]
```diff
@@ -15,14 +15,15 @@
...
           <slot name="preIcon"></slot>
         </div>
-        <slot
-          ><h4
+        <slot>
+          <h4
             :title="title"
-            class="title mr-4 text-on-surface text-opacity-high min-w-expansion-panel text-left tg-body-bold-mobile"
+            class="title mr-4 text-opacity-high min-w-expansion-panel text-left tg-body-bold-mobile"
+            :class="headingColor"
```
Comment (atharva3010) [inline src/components/ExpansionPanel.vue:22]: Can bind static and dynamic classes together, for eg:
```
:class="[headingColor, 'title mr-4 text-opacity-high min-w-expansion-panel text-left tg-body-bold-mobile']"
```

**Review by atharva3010 — CHANGES_REQUESTED** [review]
Minor code quality changes, but apart from that good job!

**Comment by atharva3010** [comment]
@bamboochow haven't tested out the UI, I'll leave that to you.

**Comment by bamboochow** [comment]
@giteshk11 

# Spelling 
"Standard" is spelled wrong here 
image

# Text Styling 
Top text should be H3 in Champagne and Limousines font 
Card Titles should be H2 in Champagne and Limousines font  
image

**Review by atharva3010 — APPROVED** [review]

**Pull request merged**